addons/spreadsheet_account/static/src/accounting_functions.js

```diff
@@ -266,9 +271,7 @@ functionRegistry.add("ODOO.BALANCE", {
 });
 
 functionRegistry.add("ODOO.FISCALYEAR.START", {
-    description: _t(
-        "Returns the starting date of the fiscal year encompassing the provided date."
-    ),
+    description: _t("Returns the starting date of the fiscal year encompassing the provided date."),
```

Comment:
useless diff

Reply:
Ah, I believe this is due to the prettier and eslint web tooling